helsinki/login/messages/messages_fi.properties

```diff
@@ -60,6 +60,6 @@ expiredActionMessage = Toimenpide on vanhentunut. Jatka kirjautumisella.
 expiredActionTokenNoSessionMessage = Toimenpide on vanhentunut
 expiredActionTokenSessionExistsMessage = Toimenpide on vanhentunut
 invalidRequestMessage = Tapahtui virhellinen pyyntö
-pageExpiredTitle = Sivu on vanhentunut
+pageExpiredTitle = Sessio on vanhentunut
```

Comment:
Istunto?

Reply:
I don't know which would be more appropriate 🤔Anniina requested "sessio",